### DISCUSSION TITLE
Phishing in all my Github.io

### DISCUSSION THREAD

**Discussion by Luuan11** [community/community#41012] [Other Feature Feedback, Questions, & Ideas] [answered]
### Phishing Alert

Hello guys,

Does anyone know why my github.io have alerts about phishing? 

I was checking the VirusTotal
https://www.virustotal.com/gui/domain/luuan11.github.io

I think i'ts about my instagram clone, but idk
https://www.virustotal.com/gui/url/e8f3c7032425be44a9a55a75bc2be1e55bc00b77ac4c257ea6371f605c5c86d6?nocache=1

Could someone help me to solve this?

**Comment by melvincwng** [comment, marked as answer]
Hi @Luuan11 👋, from your public code repositories, it seems like you have a couple of web apps/pages which are clones of various major tech companies. They are hosted on github.io, and they seem to be tagged as potential phishing websites by Google Search Console.

Please understand that Google Search Console / Google Safe Browsing crawls through the web on a daily basis and tags such clone projects as potential phishing websites, possibly due to the huge similarity/resemblance with the actual websites by the actual company. This is done to protect and prevent others on the web from accidentally visiting such websites (which may have malicious intent for some of them).

To resolve your issue, my suggestion would be to take down those clone web applications from GitHub.io. Unfortunately, you would not be able to host such clone projects on the web.

Then, login to Google Search Console, and write a ticket/email to them informing that you have unhosted those clone web apps projects. Once they verified this, it should probably resolve the phishing warning on your GitHub.io pages.

Hope this is helpful and answers your question, take care and have a good day ahead 😊

**Reply by melvincwng** [reply]
@Luuan11, it is not advisable to host clone projects that have a high degree of similarity to the actual web apps of major tech companies.

Take for example, your Instagram clone & Google clone projects - these would be easily flagged by Google Safe Browsing as potential phishing sites.

You can refer to this other [GitHub Discussions link over here](https://github.com/community/community/discussions/30261) which highlights a similar experience by another developer on GitHub, if you need more info.

Thanks.

**Reply by Luuan11** [reply]
So, if i delete this repositories about clone, it will fix ?

**Reply by melvincwng** [reply]
@Luuan11, you don't have to delete your source code. 

You just need to unpublish your web pages/web apps on GitHub Pages. Please refer to this [GitHub Documentation link](https://docs.github.com/en/pages/getting-started-with-github-pages/unpublishing-a-github-pages-site) for more information.

After unpublishing your webpages/web apps, please follow the instructions in this [Google Search Central link](https://developers.google.com/search/docs/monitor-debug/security/social-engineering?hl=en#fixing-the-problem) to resolve your issue. Refer to the section on 'Fixing the problem'. Essentially, you need to request a review from Google to remove the phishing warning on your github.io pages.

Hope this clarifies & answers your question.

Thanks, and have a good day 😄 !

**Reply by Luuan11** [reply]
For now, thank you for your attention, i will do this and I'll mark as answer, Thank you One more time
Have a nice day
